skill.json

```diff
@@ -692,7 +692,8 @@
     "systemDeps": false,
     "requirements": {
         "python": [
-            "pfzy==0.3.4"
+            "pfzy==0.3.4",
+            "neon-utils~=1.4.0"
```

Comment:
I don't think you're using `neon-utils` anywhere here, but you are using `ovos-workshop` and `ovos-utils` (logging)

Reply:
I had included neon-utils to fix some issue or other...I'll remove and see what happens